rest/snippets/PlaybackInfo/PlaybackInfoEntity.yml

```diff
@@ -0,0 +1,55 @@
+PlaybackInfoEntity:
```

Comment:
It might make sense to have separate entities for requests and responses, as their schemas might differ considerably.